docs/data-header.rst

```diff
@@ -165,8 +169,8 @@ This part of the header is represented by :class:`audformat.Split`.
 Field           Mandatory  Description
 ==============  =========  ====================================================
 id              yes        Unique identifier of split
-type            yes        Split type,
-                           typically one of :class:`audformat.define.SplitType`
```

Comment:
Doesn't make sense to refer to the class here? In case we ever extend it we have to also change it here.

Reply:
As we have the format specification here my goal was to formulate this independent of any implementation.
One consequence is of cause that we have to change it at two different places if we ever want to change it.
But I would argue those changes should happen extremely rarely ;)

Reply:
Ok :)